docs/application/native/guides/device/battery-monitor.md

```diff
@@ -67,7 +67,7 @@ To get the battery usage information related to the application ID:
        if (error_code == BATTERY_MONITOR_ERROR_NONE)
            printf("The Battery Usage for appid [%s], for resource cpu is [%lf] mAh", app_id, battery_usage);
        else if (error_code == BATTERY_MONITOR_ERROR_RECORD_NOT_FOUND)
-           printf("The Battery Usage for appid [%s], for resource cpu is not recorded");
+           printf("The Battery Usage for appid [%s], for resource cpu is not recorded, app_id");
```

Comment:
Thanks for the update @d1-agarwal.

However, would you please check the " again?
It looks like the app_id is included in the log string.